### PR TITLE
Background List Component & Default Images

### PR DESCRIPTION
**Pull request by ask7708**
- Added React component to render list of images
- Added default images for demo purposes
- Removed keyer component (was left in from proof of concept)

### CODE REVIEW

File: grr-ui/src/templates/AssetMgmt/AssetMgmt.js [hunk 1]
```diff
@@ -11,9 +12,8 @@ class AssetMgmt extends Component {
     constructor(props){
         super(props);
         this.toggleVRMode = this.toggleVRMode.bind(this);
-        this.state = {vrMode: false, video:null, width: null, height: null, canvas: null};
-
-
+        this.selectBackground = this.selectBackground.bind(this);
```
Comment (lance13c) [inline grr-ui/src/templates/AssetMgmt/AssetMgmt.js]: Is this line needed? I thought the class instance's scope is already bound to all methods and constructors in es6 classes.

File: grr-ui/src/templates/AssetMgmt/AssetMgmt.js [hunk 2]
```diff
@@ -23,6 +23,10 @@ class AssetMgmt extends Component {
 
     }
 
+    selectBackground (backgroundTitle) {
+        this.setState({selectedBackground: backgroundTitle});
```
Comment (lance13c) [inline grr-ui/src/templates/AssetMgmt/AssetMgmt.js]: Could the name "selectedBackground" be changed to "currentBackground", "background", "backgroundImage", or another name. selectedbackground was confusing to me when there is also a selectBackground method.

Don't know if this is too nit-picky.

File: grr-ui/src/templates/BackgroundImageList/BackgroundImageList.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+
+    render() {
+        return (<div className="list-container">
+            <Menu className="pt-large">
+                <li className="pt-menu-header centered-header">
+                    <h6>Default Background Options</h6>
+                </li>
+                <MenuItem text="stock360.png" onClick={this.handleClick}/>
```
Comment (lance13c) [inline grr-ui/src/templates/BackgroundImageList/BackgroundImageList.js]: The list of MenuItem and MenuDivider's is pretty repetitive. Could the list of images be in an array in a configuration file? Then import the config and image array. Loop the array and add in the MenuItems.

```javascript
return (
   <div>
    {this.props.level.map((item, index) => (
       <span className='indent' key={index} />
    ))}
    "Some text value"
   </div>
);
```

File: grr-ui/src/templates/AssetMgmt/AssetMgmt.js (hunk 1 above)
Comment (lance13c) [inline grr-ui/src/templates/AssetMgmt/AssetMgmt.js]: Is this line needed? I thought the class instance's scope is already bound to all methods and constructors in es6 classes.

Comment (ask7708) [reply grr-ui/src/templates/AssetMgmt/AssetMgmt.js]: Yeah, I thought the same initially but ran into issues when I didn't bind. I believe we need to currently do it this way or we run into scoping issues.

File: grr-ui/src/templates/BackgroundImageList/BackgroundImageList.js (hunk 3 above)
Comment (ask7708) [reply grr-ui/src/templates/BackgroundImageList/BackgroundImageList.js]: Fixed here: b7aeab8

File: grr-ui/src/templates/AssetMgmt/AssetMgmt.js (hunk 2 above)
Comment (ask7708) [reply grr-ui/src/templates/AssetMgmt/AssetMgmt.js]: Definitely understandable. Fixed here: 7b8e920

**Pull request merged**